### PR TITLE
[TypeScript] Implement endpointName parameter for Botskills

### PR DESCRIPTION
**Pull request by VictorGrycuk**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: tools/botskills/src/botskills-connect.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ program
     .description('Connect a skill to your assistant bot. Only one of both path or URL to Skill is needed.')
     .option('-l, --localManifest <path>', 'Path to local Skill Manifest file')
     .option('-r, --remoteManifest <url>', 'URL to remote Skill Manifest')
+    .option('-n, --endpointName <name>', 'Name of the endpoint to connect to your assistant (case sensitive)')
```
Comment (Batta32) [inline tools/botskills/src/botskills-connect.ts]:
```suggestion
    .option('-n, --endpointName [name]', '[OPTIONAL] Name of the endpoint to connect to your assistant (case sensitive)')
```

Comment (Batta32) [reply tools/botskills/src/botskills-connect.ts]: I'd to add a explanation if it's contains a default value or not

Comment (Batta32) [reply tools/botskills/src/botskills-connect.ts]: What do you think to change the alias `-n` to another one? I think that `-n` doesn't represent to `endpointName` argument, maybe `-e`?

File: tools/botskills/src/botskills-update.ts [hunk 2]
```diff
@@ -32,6 +32,7 @@ program
     .description('Update a specific skill from your assistant bot.')
     .option('-l, --localManifest <path>', 'Path to local Skill Manifest file')
     .option('-r, --remoteManifest <url>', 'URL to remote Skill Manifest')
+    .option('-n, --endpointName <name>', 'Name of the endpoint to connect to your assistant (case sensitive)')
```
Comment (Batta32) [inline tools/botskills/src/botskills-update.ts]: Same feedback for this argument

File: tools/botskills/src/botskills-connect.ts (hunk 1 above)
Comment (VictorGrycuk) [reply tools/botskills/src/botskills-connect.ts]: I'll work on these feedback

File: tools/botskills/src/botskills-connect.ts [hunk 3]
```diff
@@ -32,6 +32,7 @@ program
     .description('Connect a skill to your assistant bot. Only one of both path or URL to Skill is needed.')
     .option('-l, --localManifest <path>', 'Path to local Skill Manifest file')
     .option('-r, --remoteManifest <url>', 'URL to remote Skill Manifest')
+    .option('-e, --endpointName <name>', '[OPTIONAL]Name of the endpoint to connect to your assistant (case sensitive)(defaults to using the first endpoint)')
```
Comment (Batta32) [inline tools/botskills/src/botskills-connect.ts]:
```suggestion
    .option('-e, --endpointName [name]', '[OPTIONAL] Name of the endpoint to connect to your Virtual Assistant (case sensitive)(by default the tool uses the first endpoint of the manifest)')
```

File: tools/botskills/src/botskills-update.ts [hunk 4]
```diff
@@ -32,6 +32,7 @@ program
     .description('Update a specific skill from your assistant bot.')
     .option('-l, --localManifest <path>', 'Path to local Skill Manifest file')
     .option('-r, --remoteManifest <url>', 'URL to remote Skill Manifest')
+    .option('-e, --endpointName <name>', '[OPTIONAL]Name of the endpoint to connect to your assistant (case sensitive)(defaults to using the first endpoint)')
```
Comment (Batta32) [inline tools/botskills/src/botskills-update.ts]:
```suggestion
    .option('-e, --endpointName [name]', '[OPTIONAL] Name of the endpoint to connect to your Virtual Assistant (case sensitive)(by default the tool uses the first endpoint of the manifest)')
```

File: tools/botskills/src/botskills-connect.ts [hunk 5]
```diff
@@ -32,6 +32,7 @@ program
     .description('Connect a skill to your assistant bot. Only one of both path or URL to Skill is needed.')
     .option('-l, --localManifest <path>', 'Path to local Skill Manifest file')
     .option('-r, --remoteManifest <url>', 'URL to remote Skill Manifest')
+    .option('-e, --endpointName <name>', '[OPTIONAL] Name of the endpoint to connect to your assistant (case sensitive)(defaults to using the first endpoint)')
```
Comment (Batta32) [inline tools/botskills/src/botskills-connect.ts]:
```suggestion
    .option('-e, --endpointName <name>', '[OPTIONAL] Name of the endpoint to connect to your assistant (case sensitive)(by default the tool will use the first endpoint of the manifest)')
```

File: tools/botskills/src/botskills-update.ts [hunk 6]
```diff
@@ -32,6 +32,7 @@ program
     .description('Update a specific skill from your assistant bot.')
     .option('-l, --localManifest <path>', 'Path to local Skill Manifest file')
     .option('-r, --remoteManifest <url>', 'URL to remote Skill Manifest')
+    .option('-e, --endpointName <name>', '[OPTIONAL] Name of the endpoint to connect to your assistant (case sensitive)(defaults to using the first endpoint)')
```
Comment (Batta32) [inline tools/botskills/src/botskills-update.ts]:
```suggestion
    .option('-e, --endpointName <name>', '[OPTIONAL] Name of the endpoint to connect to your assistant (case sensitive)(by default the tool will use the first endpoint of the manifest)')
```

**Pull request merged**